File: reto_clean_code/angular/domain/pago.ts
// Representa un pago asociado a un pedido.  Contiene la lógica de
// procesamiento como placeholder para futuras implementaciones.

export class Pago {
  constructor(public idPedido: string, public monto: number, public medioPago: string) {}

  procesar(): void {
    // Lógica de procesamiento del pago (placeholder)
  }
}

File: reto_clean_code/python/domain/pago.py
"""
Representa un pago de un pedido.  La lógica de procesamiento se deja
como placeholder para que los participantes la implementen según sus
requerimientos.
"""


class Pago:
    def __init__(self, id_pedido: str, monto: float, medio_pago: str) -> None:
        self.id_pedido = id_pedido
        self.monto = monto
        self.medio_pago = medio_pago

    def procesar(self) -> None:
        # Lógica de procesamiento de pago (placeholder)
        pass